tajo-core/src/main/proto/InternalTypes.proto

```diff
@@ -45,3 +45,8 @@ message CorrProto {
   required double yvar = 5;
   required double covar = 6;
 }
+
+message StringAggProto {
+  required string value = 1;
```

Comment:
IMO, this proto is unnecessary define. the StringAgg is only to aggregate a value.
What do you think about that?